api/account/api.go

```diff
@@ -127,33 +129,27 @@ func ConvertToAPIAccount(ctx context.Context, userKc kc.UserRepresentation, logg
 	userRep.Username = userKc.Username
 	userRep.Email = userKc.Email
 	userRep.EmailVerified = userKc.EmailVerified
+	userRep.EmailToValidate = userKc.GetAttributeString(constants.AttrbEmailToValidate)
 	userRep.FirstName = userKc.FirstName
 	userRep.LastName = userKc.LastName
+	userRep.PhoneNumber = userKc.GetAttributeString(constants.AttrbPhoneNumber)
+	userRep.PhoneNumberToValidate = userKc.GetAttributeString(constants.AttrbPhoneNumberToValidate)
+	userRep.Gender = userKc.GetAttributeString(constants.AttrbGender)
+	userRep.Locale = userKc.GetAttributeString(constants.AttrbLocale)
+	userRep.BusinessID = userKc.GetAttributeString(constants.AttrbBusinessID)
 
-	if value := userKc.GetAttributeString(constants.AttrbPhoneNumber); value != nil {
```

Comment:
Because we don't need to check if the attribute is there or not... everything is done in GetAttributeString (already returns nil if attribute is missing) and userRep has just been created so the default value is already nil